src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -23,8 +23,8 @@
 using System.Runtime.InteropServices;
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-linux`
Invoke-WebRequest tests.Denial of service.Image Parsing
```powershell
Expected the actual value to be greater than 10, but got 0.339536922042311.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1897
1897:             $pathologicalRatio | Should -BeGreaterThan 10
```